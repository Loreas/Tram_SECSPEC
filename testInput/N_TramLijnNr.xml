<SYSTEM>
    <station>
        <naam>A</naam>
        <spoor>
            <spoorNr>12</spoorNr>
            <volgende>C</volgende>
            <vorige>C</vorige>
        </spoor>
        <type>halte</type>
    </station>
    <tram>
        <lijnNr>-12</lijnNr>
        <zitplaatsen>32</zitplaatsen>
        <snelheid>50</snelheid>
        <beginStation>A</beginStation>
        <type>PCC</type>
        <voertuigNr>150</voertuigNr>
    </tram>
    <passagier>
        <naam>A</naam>
        <beginstation>A</beginstation>
        <eindstation>B</eindstation>
        <hoeveelheid>10</hoeveelheid>
    </passagier>
</SYSTEM>
