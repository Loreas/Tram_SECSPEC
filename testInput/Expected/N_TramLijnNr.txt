======================
Starting XML Parser...
======================

Creating new Station:
---------------------
	Getting Station name...
		Station Name: A
	Getting Station Track...
		Track Number: 12
	Getting Next Station...
		Next Station: A
	Getting Previous Station...
		Previous Station: A
	Getting Station Type...
		Station Type: halte

Creating new Tram:
------------------
	Getting Tram Number...
		FAULT IN TRAM LINE NUMBER.
	Getting Tram seats...
		Tram Seats: 32
	Getting Tram speed...
		Tram Speed: 50
	Getting Starting Station...
		Starting Station: A
	Getting Current Station...
		Current Station: A
	Getting Tram Type...
		Tram Type: PCC
	Getting Tram Vehicle Number...
		Tram Vehicle Number: 150
DELETING INCOMPLETE TRAM

Creating new Passagier:
-----------------------
	Getting Passenger Name...
		Passenger Name: A
	Getting Passenger Starting Station...
		Starting Station: A
	Getting Passenger End Station...
		End Station: B
	Getting Amount of Passengers...
		Passenger Amount: 10
===================
XML Parser finished
===================

